nni/algorithms/compression/v2/pytorch/base/pruner.py

```diff
@@ -125,54 +128,3 @@ def show_pruned_weights(self, dim: int = 0):
                 sum_idx.remove(dim)
                 index = torch.nonzero(weight_mask.abs().sum(sum_idx) != 0, as_tuple=False).tolist()
             _logger.info(f'simulated prune {wrapper.name} remain/total: {len(index)}/{weight_mask.size(dim)}')
-
-    def export_model(self, model_path, mask_path=None, onnx_path=None, input_shape=None, device=None):
```

Comment:
export is removed, why?

Reply:
Because I think the old `export_model()` is useless now because we return masks directly in `compress()`, maybe let users handle export logic is better, or we can support a new `export_model()`.

Reply:
After removing the exporting function, what's the new interface between the pruner and speedup?

Reply:
revert `export_model`, and remove the onnx export.